### ISSUE TITLE
Capacitor 2.0.0 / AndroidX support

### ISSUE THREAD

**Issue by disbelief** [jepiqueau/capacitor-data-storage-sqlite#17]
With Capacitor 2.0.0 coming — which switches to AndroidX — the `capacitor-data-storage-sqlite` plugin will need to be updated to continue functioning.

I've tried using [jetifier](https://www.npmjs.com/package/jetifier) to manually update the plugin locally, but my Capacitor app isn't finding the `CapacitorDataStorageSqlite` class after doing so:

```
error: cannot find symbol class CapacitorDataStorageSqlite
```

It may be as simple as using Android Studio's `Refactor` > `Migrate to AndroidX` function on the source project. I've tried it with the source code that's installed in `node_modules/capacitor-data-storage-sqlite/android/` but it doesn't seem to work (I'm guessing simply due to Android Studio project configuration missing).

I imagine you'll be seeing a lot of requests for this shortly because the latest version of Xcode (11.4) breaks Capacitor version `1.x` so people are starting to upgrade to Capacitor version `2.0.0-beta.1`

**Comment by disbelief** [comment]
I whipped up a PR for this #18

**Comment by jepiqueau** [comment]
@disbelief Thanks for your comment, I noticed that when upgrading to Xcode 11.4 i could not recompile my capacitor projects and as i was working on updating my web components for the new stencil 1.11.2 and having quite a lot of issues i was telling to myself that i would wait capacitor 2 to do something but i may have a look earlier to the beta release

**Comment by disbelief** [comment]
For what it's worth I'm running the beta release and haven't noticed any issues. I've been waiting for 2.0.0 for quite some time, but there doesn't still appear to be a release date set. Who knows when it'll be merged, but maybe this Xcode nonsense will drive the capacitor devs to speed it up a bit?

**Comment by disbelief** [comment]
Looks like [capacitor version 2.0.0 has officially been released](https://github.com/ionic-team/capacitor/releases/tag/2.0.0)

**Comment by disbelief** [comment]
I submitted a new PR with the changes I made for capacitor 2.0.0 support, if you're interested. It is working for me in my app currently: #19

**Comment by bleuscyther** [comment]
May change :
````java
import com.jeep.plugins.capacitor.CapacitorDataStorageSqlite;
```

to 

```java
import com.jeep.plugin.capacitor.CapacitorDataStorageSqlite;
```
in the **MainActivity** file

**Comment by disbelief** [comment]
@bleuscyther I fixed those typos in the Readme (e0dbd76) let me know if there's anything else.

**Comment by bleuscyther** [comment]
Everything is good so far, i am using your pull request since yesterday

**Comment by jepiqueau** [comment]
@disbelief This is now included in the last release 2.0.0-1 of capacitor-data-storage-sqlite. Thanks for having provided in advance a pull request for other users. I have now close the pull request and will close this issue. Many thanks for using the plugin
